ui/src/app/archived-workflows/components/archived-workflow-list/archived-workflow-list.tsx

```diff
@@ -57,38 +62,110 @@ export class ArchivedWorkflowList extends BasePage<RouteComponentProps<any>, Sta
             throw this.state.error;
         }
         return (
-            <Page
-                title='Archived Workflows'
-                toolbar={{
-                    breadcrumbs: [{title: 'Archived Workflows', path: uiUrl('archived-workflow')}],
-                    tools: [
-                        <NamespaceFilter
-                            key='namespace-filter'
-                            value={this.namespace}
-                            onChange={namespace => {
-                                this.namespace = namespace;
-                            }}
-                        />
-                    ]
-                }}>
-                <div className='row'>
-                    <div className='columns small-12'>{this.renderWorkflows()}</div>
-                </div>
-            </Page>
+            <Consumer>
+                {ctx => (
+                    <Page
+                        title='Archived Workflows'
+                        toolbar={{
+                            breadcrumbs: [{title: 'Archived Workflows', path: uiUrl('archived-workflows')}],
+                            actionMenu: {
```

Comment:
Add a `Submit New Workflow` button on this page. This workarounds the unclear `Page/toolbar` bug.

More information about the bug mentioned above - When there's `actionMenu` or `tools` in `toolbar`, or `toolbar` is removed, the page will get `too much recursion` error when making several changes on the `namespace` or `label` field. A rough investigation shows there might be some conflict between `Page/toolbar` and `Autocomplete`.

Add a `Submit New WF` button here to bypass it for now and look into the issue further later.